extra_data/reader.py

```diff
@@ -2131,11 +2133,33 @@ def open_run(
     if isinstance(aliases, str):
         aliases = Path(aliases.format(prop_dir))
 
-    # If we're using the default aliases file and it doesn't exist, ignore it
-    # without throwing any errors.
+    # If we're using the default aliases file and it doesn't exist, try to
+    # bootstrap it from an instrument template before falling back to no
+    # aliases.
     default_aliases = Path(DEFAULT_ALIASES_FILE.format(prop_dir))
     if aliases == default_aliases and not default_aliases.is_file():
-        aliases = None
+        # Determine the instrument from the proposal path
+        instrument = None
+        template_path = None
+        if instrument := Path(prop_dir).parts[-3]:
```

Comment:
I don't think this if-statement is necessary, `.parts` is just a tuple so indexing it will either succeed and give a path segment (from the docs I don't think it'll ever return an empty string) or throw an out-of-bounds error (which would arguably be the users fault for passing in a bad path).